src/test/java/dog/pawbook/logic/commands/CommandTestUtil.java

```diff
@@ -65,10 +65,10 @@
     public static final String VALID_TAG_QUIET = "quiet";
     public static final String VALID_ENTITY_ID = "1";
 
-    public static final String VALID_NAME_FIRSTPROGRAM = "Basic Obedience Training";
-    public static final String VALID_NAME_SECONDPROGRAM = "Basic Behavioural Training";
-    public static final String VALID_DOG_ID_1 = "1";
-    public static final String VALID_DOG_ID_2 = "2";
+    public static final String VALID_NAME_OBEDIENCE = "Basic Obedience Training";
+    public static final String VALID_NAME_BEHAVIOURAL = "Basic Behavioural Training";
+    public static final int VALID_DOG_ID_1 = 1;
+    public static final int VALID_DOG_ID_2 = 2;
```

Comment:
I would say avoid defining dog IDs this way, especially since none of the others entities have defined their IDs this way. ID is largely dynamic and only because of the typical database being largely static that's why it was hardcoded. For tests that require the dogs IDs to be present but not necessarily correct, I suggest using the TypicalId class and use some of the numbers inside.